.github/workflows/syncNuFiConnectDocs.yml

```diff
@@ -0,0 +1,17 @@
+name: Sync docs with gitbook
+
+on:
+  push:
+    branches: ['github-actions-readme']
+    paths: ['docs/cardanoNuFiConnect.md']
+
+jobs:
+  copy_nufi_connect_docs:
+    name: Copy Cardano NuFi Connect Docs
+    uses: ./.github/workflows/copyToGitbookRepo.yml
+    with:
+      source_file: 'docs/cardanoNuFiConnect.md'
+      rename: 'nufi-dapp-sdk-for-cardano.md'
+      destination_folder: 'sso-nufi-dapp-sdk-for-cardano'
+    secrets:
+      access_token: ${{ secrets.API_TOKEN_GITHUB }}
```

Comment:
This token is stored inside repository settings. I've generated this token for NuFi Organization